package.json

```diff
@@ -7,20 +7,32 @@
   "author": "Michał Pierzchała <thymikee@gmail.com>",
   "license": "MIT",
   "dependencies": {
-    "@types/jest": "^23.3.1",
+    "@types/jest": "^23.3.5",
```

Comment:
since this one is in this PR, would you mind put this to `23.x` and `ts-jest` to `23.10.5` ? I know `23.x` works but `ts-jest` `23.10.x` I'm not sure it works